bootstrap/common/sync_third_party.sh

```diff
@@ -55,7 +55,8 @@ function sync_third_party() {
 
   # Install a trap to remove temporary path if this script is finished.
   # The trap will work well when unknown breaking happens like ctrl+c.
-  trap "rm -rf $temp_path" EXIT
+  export TRAP_COMMAND="rm -rf $temp_path;$TRAP_COMMAND"
```

Comment:
This is for remove all C:/tmp_@@@ directory. 
sync_third_party( ) function call twice. one is for node, another is for mongo.
but only one tmp directory remove. because when called second trap command, trap remove first one and apply second command. 
So I fix it by using env value.